### DISCUSSION TITLE
Openradioss IMPLICIT

### DISCUSSION THREAD

**Discussion by CinekPoland** [OpenRadioss/OpenRadioss#1056] [Q&A] [answered]
Hi. I have some questions
"As mentioned before, Radioss uses only the elastic part of the material parameters to build the stiffness matrix which allows working with all available Radioss materials."  does it mean that it is not possible to simulate plastic deformations using Implicit solver of Radioss ? 

Is SPH method available for implicit solver of Radioss ? 
is there no problem with simulating the phenomenon of friction (e.g. with deformation of metals caused by a tool) using sph?

**Comment by MarianBulla** [comment, marked as answer]
Hi,
Yes it is possible to simulate deformations using Implicite solver of Radioss. It will consider plastification.
You can even add failure criteria, to see, when elements are failing. But be aware that if a piece of a structure becomes separated, then the solver stopps.
No, SPH is not possible in implicit solver. Only in the explicit solution.
Friction with SPH should be possible.
Hope it helps you ?

**Reply by CinekPoland** [reply]
Thank you for you advices. Yes your answer helped me.
